src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/SessionTemplateTime.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.visitscheduler.controller.validators.SessionTimeValidation
+import java.time.LocalTime
+
+class SessionTemplateTime(
```

Comment:
Is it a slot time, SessionTimeSlot,.. I don't know your call, moving this into a a sub object is going to mess up the UI, but I agree we need it